lib/ocg_schemas/Cargo.toml

```diff
@@ -23,6 +23,7 @@ bytemuck.workspace = true
 bitflags.workspace = true
 bitvec.workspace = true
 bevy_math.workspace = true
+bevy.workspace = true
```

Comment:
bevy should definitely not go into the schemas library, it's supposed to be for data format manipulation only